pkgs/by-name/dr/dragmap/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  boost,
+  cmake,
+  gtest,
+  zlib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "DRAGMAP";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = "DRAGMAP";
+    rev = finalAttrs.version;
+    fetchSubmodules = true;
+    hash = "sha256-f1jsOErriS1I/iUS4CzJ3+Dz8SMUve/ccb3KaE+L7U8=";
+  };
+
+  nativebuildInputs = [ boost ];
+  buildInputs = [
+    gtest
+    zlib
+  ];
+
+  # Add missing include cstdint
+  # Upstream does not accept PR. Issue opened at
+  # https://github.com/Illumina/DRAGMAP/issues/63
+  patches = [ ./cstdint.patch ];
+
+  GTEST_INCLUDEDIR = "${gtest.dev}/include";
+  CPPFLAGS = "-I ${boost.dev}/include";
+  LDFLAGS = "-L ${boost.out}/lib";
+
+  installPhase = ''
```

Comment:
Shouldn't this be the `postInstall` ?

Reply:
I got no feedback on this so far.

Reply:
Hi, thanks for the follow-up. All this time, my answers were marked as pending so it looks like you never saw it !
Updating the previous answer: if you are asking about the FLAGS, they are needed for compilation. The part in installPhase cannot be done in postInstall (as it tries to install things in /usr/bin).